sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md

```diff
@@ -21,31 +21,8 @@ Another sample [azure-spring-boot-sample-active-directory-resource-server] will
 ### Environment checklist
 We need to ensure that this [environment checklist][ready-to-run-checklist] is completed before the run.
 
-### Include the package
-```xml
-<dependencies>
-    <dependency>
-      <groupId>com.azure.spring</groupId>
-      <artifactId>azure-spring-boot-starter-active-directory</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-oauth2-client</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-web</artifactId>
-    </dependency>
-</dependencies>
-```
-
 ### Configure your middle-tier Web API A
 1. Expose two scopes for ***Web API A***, `Obo.Graph.Read` and `Obo.File.Read`. The **Application ID URI** is generated by default.  
-   **NOTE**: Theses scopes (`Obo.Graph.Read`, `Obo.File.Read`) should be added to the **Web APP**. 
```

Comment:
Why are we removing this?

Reply:
For simplicity, remoted. Current scopes of the resource server can be added by Web App or Resource Server, not just Web App.